Formula/sratoolkit.rb

```diff
@@ -0,0 +1,72 @@
+class Sratoolkit < Formula
+  desc "Data tools for INSDC Sequence Read Archive"
+  homepage "https://github.com/ncbi/sra-tools"
+  url "https://github.com/ncbi/sra-tools/archive/2.8.2-3.tar.gz"
+  version "2.8.2-3"
+  sha256 "d04e5fd735c3987c3497369844e20c1ae38a88867c36154798f072f19a8792fa"
+  head "https://github.com/ncbi/sra-tools.git"
+
+  depends_on "hdf5"
+  depends_on "libmagic"
+  depends_on "libxml2"
+
+  resource "ngs-sdk" do
+    url "https://github.com/ncbi/ngs/archive/1.3.0.tar.gz"
+    sha256 "803c650a6de5bb38231d9ced7587f3ab788b415cac04b0ef4152546b18713ef2"
+  end
+
+  resource "ncbi-vdb" do
+    url "https://github.com/ncbi/ncbi-vdb/archive/2.8.2-2.tar.gz"
+    version "2.8.2-2"
+    sha256 "7866f7abf00e35faaa58eb3cdc14785e6d42bde515de4bb3388757eb0c8f3c95"
+  end
+
+  def install
+    resource("ngs-sdk").stage do
+      cd "ngs-sdk" do
+        system "./configure",
+          "--prefix=#{prefix}",
+          "--build=#{Pathname.pwd}/ngs-sdk-build"
+        system "make"
+        system "make", "install"
+      end
+    end
+
+    (buildpath/"ncbi-vdb").install resource("ncbi-vdb")
+    cd "ncbi-vdb" do
+      system "./configure",
+        "--prefix=#{prefix}",
+        "--with-ngs-sdk-prefix=#{prefix}",
+        "--build=#{buildpath}/ncbi-vdb-build"
+      ENV.deparallelize { system "make" }
+      system "make", "install"
+    end
+
+    # Fix the error: ld: library not found for -lmagic-static
```

Comment:
This is a bug that needs to be fixed upstream.

Reply:
Reported upstream: https://github.com/ncbi/sra-tools/pull/105